doc/whats_new/v1.3.rst

```diff
@@ -287,6 +287,10 @@ Changelog
   scikit-learn 1.3: retraining with scikit-learn 1.3 is required.
   :pr:`25186` by :user:`Felipe Breve Siola <fsiola>`.
 
+- |Efficiency| :class:`ensemble.BaseForest` now only recomputes out-of-bag scores
+  if `n_more_estimators > 0` in subsequent `fit` calls.
```

Comment:
`BaseForest` is not a public class and `n_more_estimators` is not a public parameter. The user guide is public facing, so it's best to state the change in terms of the public API:

```suggestion
- |Efficiency| :class:`ensemble.RandomForestClassifier` and 
  :class:`ensemble.RandomForestRegressor` with `warm_start=True` now only
  recomputes out-of-bag scores when there are actually more `n_estimators`
  in subsequent `fit` calls.
```